inspirehep/modules/authors/views.py

```diff
@@ -154,6 +154,96 @@ def get_inspire_url(data):
     return url
 
 
+@blueprint.route('/publications', methods=['GET'])
+def get_publications():
+    from elasticsearch.helpers import scan
+    from invenio_search import current_search_client
+
+    recid = request.values.get('recid', 0, type=int)
+
+    publications = []
+    collaborations = set()
+    keywords = set()
+
+    for result in scan(
+            current_search_client,
+            query={
+                '_source': ['accelerator_experiments',
+                            'control_number',
+                            'earliest_date',
+                            'facet_inspire_doc_type',
+                            'publication_info',
+                            'titles',
+                            'thesaurus_terms'
+                            ],
+                'query': {"match": {"authors.recid": recid}}
+            },
+            index='records-hep',
+            doc_type='hep'):
+
+        try:
+            result_source = result['_source']
+            publication = {}
+
+            # Get publication title (required).
+            publication['title'] = result_source.get(
```

Comment:
You can use https://github.com/inspirehep/inspire-next/blob/master/inspirehep/utils/record.py#L30 so that we unify how to get a title everywhere.